playground/backend/internal/code_processing/code_processing.go

```diff
@@ -112,6 +114,7 @@ func runStep(ctx context.Context, cacheService cache.Cache, paths *fs_tool.LifeC
 	var runError bytes.Buffer
 	runOutput := streaming.RunOutputWriter{Ctx: pipelineLifeCycleCtx, CacheService: cacheService, PipelineId: pipelineId}
 	go readLogFile(pipelineLifeCycleCtx, ctx, cacheService, paths.AbsoluteLogFilePath, pipelineId, stopReadLogsChannel, finishReadLogsChannel)
+	go readGraphFile(pipelineLifeCycleCtx, ctx, cacheService, filepath.Join(paths.AbsoluteBaseFolderPath, preparers.GraphFileName), pipelineId)
```

Comment:
`LifeCyclePaths` contains value `AbsoluteLogFilePath`. Maybe we can do the same for the graph file instead of `filepath.Join(paths.AbsoluteBaseFolderPath, preparers.GraphFileName)`?

Reply:
Done